bin.js

```diff
@@ -64,8 +64,14 @@ if (opts['upload-all']) {
 }
 
 function uploadFiles (files) {
-  buildLog('Uploading ' + files.length + ' prebuilds(s) to Github releases')
-  upload(extend(opts, {files: files}), function (err, result) {
+  // NOTE(robinwassen): Only include unique files
+  // See: https://github.com/prebuild/prebuild/issues/221
+  var uniqueFiles = files.filter(function (file, index) {
+    return files.indexOf(file) === index
```

Comment:
Shouldn't that be `files.lastIndexOf`?

Reply:
Maybe we should just use `lodash.uniq`

Reply:
Nevermind, I see why `indexOf` also works